src/testing/code-debugging.md

```diff
@@ -17,7 +17,7 @@ For a full list of debugging and profiling tools, see the
 
 [`flutter_gdb`]: https://github.com/flutter/engine/blob/main/sky/tools/flutter_gdb
 
-## Logging
+## Log your application
```

Comment:
Should this be "Add logging to your application"?

I don't think of log as a verb outside of tree felling...